data/cameras.xml

```diff
@@ -18193,6 +18193,9 @@
 			<Alias id="GR IIIx HDF">RICOH GR IIIx HDF</Alias>
 		</Aliases>
 	</Camera>
+	<Camera make="RICOH IMAGING COMPANY, LTD." model="RICOH GR IV" mode="dng" supported="unknown-no-samples">
```

Comment:
I think this just shouldn't have a `supported` attr,
as per precedent because it is a DNG.